mobile/src/filtersModal.js

```diff
@@ -73,7 +73,20 @@ export default class FilterSelect extends Component {
     }
   }
 
-
+  returnTotal = (sort) => {
+    const {questions, session} = this.props
+    let orderedQuestions = questions
+    switch (sort) {
+      case "Answered" : orderedQuestions = questions.filter(item => item.answered && item.session === session.key)
+      return orderedQuestions.length
```

Comment:
Typically, everything under a `case` is indented so it's easy to see what is executed in that `case`